common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp

```diff
@@ -201,20 +145,28 @@ class TrtCommon  // NOLINT
   TrtUniquePtr<nvinfer1::ICudaEngine> engine_;
   TrtUniquePtr<nvinfer1::IExecutionContext> context_;
   std::unique_ptr<nvinfer1::IInt8Calibrator> calibrator_;
-
   nvinfer1::Dims input_dims_;
   nvinfer1::Dims output_dims_;
   std::string precision_;
   BatchConfig batch_config_;
   size_t max_workspace_size_;
   bool is_initialized_{false};
-
   // profiler for per-layer
-  SimpleProfiler model_profiler_;
+  SimpleProfiler m_model_profiler;
   // profiler for whole model
-  SimpleProfiler host_profiler_;
-
-  std::unique_ptr<const BuildConfig> build_config_;
+  SimpleProfiler m_host_profiler;
+  // type for calibration
+  nvinfer1::CalibrationAlgoType m_calibType;
+  // flg for dla
+  int m_dla;
+  // flg for partial quanitzation in first layer
```

Comment:
```suggestion
  // flg for partial quantization in first layer
```

Reply:
fixed in commit [d1a2ec5](https://github.com/autowarefoundation/autoware.universe/pull/4125/commits/d1a2ec58dee7ade4b4b0cf2c582817e88c1122dd)